python/eos/tasks.py

```diff
@@ -476,8 +476,11 @@ def corner_plot(analysis_file:str, posterior:str, base_directory:str='./'):
     :type posterior: str
     :param base_directory: The base directory for the storage of data files. Can also be set via the EOS_BASE_DIRECTORY environment variable.
     :type base_directory: str, optional
+    :param format: The file extension of the data files. Can also be a list of file extensions.
+    :type format: str or list, optional
```

Comment:
This requires a specification of what values ``format`` or its elements may take. A referal to the matplotlib's documentation and output formats would suffice I think.

Reply:
Also, ``str or list of str`` is a better description.